### PR TITLE
UCT/UD: Fix timer backoff initialization/check

### PR DESCRIPTION
**Pull request by brminich**
## What
Fix "SLOW_TIMER_BACKOFF" check

## Why ?
UD back off is slowtimer multiplier, defining it less than 1 makes no sense 

Currently defining it to small values give the following (unclear) fatal error:
`timer_wheel.c:63   Fatal: Timer resolution is too low. Min resolution 25811.101538 usec, wanted 20971.520000 usec`

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12349/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 20-Sep-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/94e1ca255d5d5b990463f729a5e379f08b2d8886/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/16d7ca2fee2cba11a209d5ef20348b8d28372606/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/6dc13680221d1e19160ed2bf844cef2dd5935d0b/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/68a7c7a5df3ce8b8c31f759e230f7e0624c23c18/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/19f94e90e2f260cf4674f236259baadebf4dea56/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/e5bf4a22d3146a1a6e54adad1e54df2ad934abc1/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/1bc1f8a5fbf9d090a52c6d6c267fa6a447ceac26/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/24f2134c8d94ebb3b60fd48fe00fde36c465bec6/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/559732fcb6f560cf0841024928761d33b5c16aa4/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/7f88cc4be7e9d456643cb26f6584ab6541147f37/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/dec6ddf1ff0bd2795b26f18b02748f485e1a4270/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/cc8aa489047d8a05c308413fa800ccda3b81887b/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/b60089342c76533c092d775908444e2ec0218a95/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/4f799a01be8de2041f7edcfdce11b0d705da0d9e/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/ce3e0a5b1d2752650fae3f006b0fbdc25f9d05ab/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/844ac3f7480979de061b92f4b399cc858d1af2a4/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/4dca2a9dc183faadc671db2cb9b8af7cb6c63547/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/13256664cde681acb307ad5bb73307f0d2893af5/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/7d9c3c6908447dedfc39f93c0c8b5b35bb9e1c79/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/966954618f60944613c01e6035ca7d16c97dc7e9/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/3ce5704fe6ab7957c4ee96de6ab78d024c7b19ae/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/d4cc7b88228dbec6501f25308df11c890de6d77b/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/dc37f0f74305dea787daa78598afe9034937cd78/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/02b5a6b0bbaf41d525da8e12554d3a3a9adf2f6a/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/a24b6bba67b96ea1eaf71c88bd6aa675c6ec1380/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/4dd03025c3205cdd491a5a8a803bc3f89b82ed33/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/a9cf816798311f56e777e8dc0532614e85706869/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/7cea940bd31dea3a7230ec97e845454a2939e605/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/f4ef78f5f7640cdb4d05a99531e9077c/raw/2fce9ae7d50a779b7209275cf45a1304fe690e30/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by brminich** [comment]
@yosefe, plz review

File: src/uct/ib/ud/base/ud_iface.c [hunk 1]
```diff
@@ -449,8 +449,8 @@ UCS_CLASS_INIT_FUNC(uct_ud_iface_t, uct_ud_iface_ops_t *ops, uct_md_h md,
         self->async.slow_tick = ucs_time_from_sec(config->slow_timer_tick);
     }
 
-    if (config->slow_timer_backoff <= 0.) {
-        ucs_error("The slow timer back off should be > 0 (%lf)",
+    if (config->slow_timer_backoff < 1.0) {
```
[the remark this answers is not in the document]
Comment (brminich) [reply src/uct/ib/ud/base/ud_iface.c]: Would need to add func/macro for adding it to uct_ud_iface_config_table. Imo better keep it as it is now - this value is logical and not supposed to be changed.

[the remark this answers is not in the document]
Comment (yosefe) [reply src/uct/ib/ud/base/ud_iface.c]: why need to add macro?
can we define UCT_UD_MIN_TIMER_TIMER_BACKOFF 1.0
then 
```
"Timeout multiplier for resending trigger (must be >= " UCS_PP_QUOTE(UCT_UD_MIN_TIMER_TIMER_BACKOFF) ")",
```

and
```
ucs_error("The slow timer back off must be >= %lf (%lf)", UCT_UD_MIN_TIMER_BACKOFF
```

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12355/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 23-Sep-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/3d4c24a7edb392d63166e62ba613e378eb37fe92/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/3996ebc6a3741c3e236ad58056333c16bd368ce3/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/205453d3e82c07264a4ac490689ee2394c9fb708/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/0975f2683997b99a4c9d4e7edcae7e640294b774/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/968fc0b01b422da783e44fafce4a5e0dde55c44d/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/3f78d5806b100d47ec4e84d2a66e0eebbb257a9b/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/63a7eeabdcef0f8485e889866a5b4f23e524e6c5/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/3103fabefc7e7c4f3a6ac51d1c169144385b42c4/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/26b9e8c557b1b947dee17015f2f8184cf8906493/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/bb01f9815fc565c94ecb602b902b103a90090357/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/5d485726aec708ecb84ad3dad178ba01316b1502/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/2a35cb715adb75053d1743402788e0ca0937e63f/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/ed705e31c792baf21d8e2ffb69bdf1ec4003df38/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/b3e32d0a817a9b24f05e20977fec1d874229277b/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/08e62f438a791166609074a2eb5bb265032bd02c/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/2e5aa5f29ae31d83dfc3e8ea8947bea5e75a871f/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/4dfe2959b8b1c34268e6afc7f63d3554ccb13376/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/d5bcbba8ff7f6414381d74fb12afb5379814c5cc/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/e4e3844a70bdc1bff1d0ea6710be3cb8170f862d/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/89c050a81f4f749c3d0217d455de239c3b9734b6/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/090ef690585d65fcd0ed5897ef535187b13f2d52/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/f775e897660a69c7a5df2f403052507e7732d9f6/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/a1b171925710d996f2ce838740d332c79cfab97f/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/c36d856b059b26244fc5414960bf2e5a079b2188/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/ba4f8b998254c80860e66d75ed2856948dee70f7/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/f98ba733433761bdebeeaa435d8c1d62651d4043/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/70a7bd7a23231771d68d0814d1b0615d72c6ae94/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/3fc4bcb55185ea94eb9019a24060adee0fb27ea0/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/519a0f9d066d584ddfaf607b72e938da/raw/aba9af8c56b70f0a902a19659366192bb9f55be0/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by brminich** [comment]
@yosefe, pls review

**Pull request merged**